pcl_ransac/include/pcl_ransac/pcl_ransac.hpp

```diff
@@ -5,25 +5,27 @@
 #include <pcl/sample_consensus/ransac.h>
 #include <pcl/sample_consensus/sac_model_plane.h>
 #include <pcl/sample_consensus/sac_model_sphere.h>
-
-#include <Eigen/Core>
-
 #include <pcl_ransac_msgs/ModelCoefficients.h>
+#include <Eigen/Core>
 
 namespace ariitk::pcl_ransac {
 
 class PointCloudModel {
     public:
         PointCloudModel();
         ~PointCloudModel() {};
+        
         virtual void generatePointCloud(pcl::PointCloud<pcl::PointXYZ>::Ptr& cloud) = 0;
-        inline std::vector<int> getModelInliers() const { return inliers_; };
         virtual void fitModel(pcl::PointCloud<pcl::PointXYZ>::Ptr& cloud, const double& distance_threshold) = 0;
+        
```

Comment:
@gajena Are we supposed to leave spaces between functions?

Reply:
Depends, usually, I don't leave out a line, but maybe you want to group certain function.